plugins/log.js

```diff
@@ -1,39 +1,19 @@
 const fs = require("fs");
-const logFile = fs.createWriteStream("screeps.log", { flags: "a" });
-const errorLogFile = fs.createWriteStream("screeps.errors.log", { flags: "a" });
 
 module.exports = function(multimeter) {
-  let enabled = multimeter.config.log === "true";
+  let log = multimeter.config.logFilename;
+  if (!log) return;
```

Comment:
What's this for? What if I only want to see errors in my log (which I think is probably the "normal" case for long-running scripts not in active development).